Packs/ThycoticDSV/pack_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "ThycoticDSV",
+    "description": "Thycotic DevOps Secrets Vault",
```

Comment:
This description doesn't add information at the moment, it should be a short overview of the pack

Reply:
@guykeller 
Fixed